config/develop/nextflow-ecs-cluster.yaml

```diff
@@ -1,13 +1,18 @@
-template_path: nextflow-ecs-cluster.yaml
+template_path: nextflow-ecs-cluster.j2
 stack_name: nextflow-ecs-cluster
 dependencies:
   - develop/nextflow-vpc.yaml
 parameters:
-  VpcId: !stack_output_external nextflow-vpc::VPCId
+  EcsSecurityGroupId: !stack_output_external nextflow-ecs-security-group::SecurityGroupId
   SubnetIds:
     - !stack_output_external nextflow-vpc::PublicSubnet
-  SecurityIngressFromPort: '80'
-  SecurityIngressToPort: '8080'
+sceptre_user_data:
+  TowerConfigFileContents: |
+    tower:
+      auth:
+        github:
+          allow-list:
+            - tess.thyer@gmail.com
```

Comment:
This list will be built out more later -- this configuration does not appear to be taking hold. Waiting to hear back from support about this.